src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -646,6 +646,77 @@ By placing a few extra blocks and changing their layout, you can create interest
 
 image::compose-configuration/page-builder-example.png[role="data-zoomable"]
 
+[#copying-list-pages]
+== Copying list pages
+
+The option to save a list page as copy allows us to create a new page that contains the same page blocks as another list page.
+This is very useful because instead of re-adding the same page blocks from another page, they can be copied simply by pressing a button.
+
+To copy the contents of another list page, navigate to the page itself and click on the btn:[Save as copy].
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/page-save-as-copy-button.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 535,
+    "w": 1920,
+    "h": 555
+  },
+  "focus": {
+    "x": 0,
+    "y": 1000,
+    "w": 1920,
+    "h": 60
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 1440,
+    "y": 1017,
+    "w": 165,
+    "h": 45
+  }]
+}
+----
+
+A new page with the same contents will be added to the list of pages.
+The structure of the new page's name will be `Copy of <name of the page it was copied from> (<number of copies>)`.
+For example, if the page that was copied from is called `Leads`, then the newly created page will be `Copy of Leads (1)`.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "compose-configuration/copied-page.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 0,
+    "y": 0,
+    "w": 315,
+    "h": 475
+  },
+  "annotations": [{
+    "kind": "box",
+    "x": 30,
+    "y": 370,
+    "w": 95,
+    "h": 15
+  }]
+}
+----
+
+[IMPORTANT]
+====
+Only page lists can be copied. Record pages cannot be saved as copies.
+====
+
```

Comment:
I'd move this block to line 654 primarily because it can get overlooked right after images